README.md

```diff
@@ -54,6 +54,15 @@ The Java API Client project is available from Duo Security on Maven.  Include th
 
 See https://mvnrepository.com/artifact/com.duosecurity/duo-client/0.3.0 for more details.
 
+# Using the Example
+There is an example in $ duo-example-admin
```

Comment:
I think the '$' here might be unnecessary.. maybe
".. example in the duo-example-admin directory"